pkg/monitoring/alloy/monitoring-config.go

```diff
@@ -0,0 +1,13 @@
+package alloy
+
+type MonitoringConfig struct {
+	Alloy MonitoringConfigAlloy `json:"alloy"`
+}
+
+type MonitoringConfigAlloy struct {
+	Controller MonitoringConfigAlloyController `json:"controller"`
+}
+
+type MonitoringConfigAlloyController struct {
```

Comment:
```suggestion
type AlloyController struct {
```

Reply:
Also do they need to be publically exposed?

Reply:
No they don't need to be public